### ISSUE TITLE
table prefixes are not working

### ISSUE THREAD

**Issue by mmmgigi** [SchumacherFM/Magento-FastIndexer#2]
Hi,

It looks like the table prefixes are not supported yet :)
For example I have the following error when I execute "php indexer.php --reindex catalog_category_flat" :

Base table or view not found: 1146 Table 'magento_test.catalog_category_entity' doesn't exist' in /www/magento_test/lib/Zend/Db/Statement/Pdo.php:228

My db table prefix is "prefix_" .

Question:
What file(s) I have to edit and manually add my prefix value? 

Thank you for your great addon!

**Comment by SchumacherFM** [comment]
Yes that is true that table prefixes have not been properly implement.
I currently do not have the time to take a deeper look :-(
